docs/product-administration-and-configuration-guide/src/main/asciidoc/kie-server-bootstrap-switches-ref.adoc

```diff
@@ -0,0 +1,246 @@
+[id='kie-server-bootstrap-switches-ref']
+= Bootstrap Switches
+
+The {KIE_SERVER} accepts a number of bootstrap switches (system properties) to configure the behavior of the server.
+
+.Bootstrap Switches for Disabling {KIE_SERVER} Extensions
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.drools.server.ext.disabled`
+|`true`, `false`
+|`false`
+|If set to `true`, disables the BRM support (for example rules support).
+
+|`org.jbpm.server.ext.disabled`
+|`true`, `false`
+|`false`
+|If set to `true`, disables the BPM support (for example processes support).
+
+|`org.optaplanner.server.ext.disabled`
+|`true`, `false`
+|`false`
+|If set to `true`, disables the BRP support (for example planner support).
+
+|`org.jbpm.ui.server.ext.disabled`
+|`true`, `false`
+|`false`
+|If set to `true`, disables the {KIE_SERVER} UI extension.
+
+|`org.kie.executor.disabled`
+|`true`, `false`
+|`false`
+|Disables the {PRODUCT} executor.
+|===
+
+NOTE: Some controller properties listed below are marked as required. Set these properties when you handle {KIE_SERVER} container creation and removal in Business Central.  If you use the {KIE_SERVER} separately without any interaction with Business Central, the properties do not have to be set.
+
+.Bootstrap Switches Required for Using a Controller
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.kie.server.id`
+|String
+|N/A
+|An arbitrary ID to be assigned to the server. If a remote controller is configured, this is the ID under which the server will connect to the controller to fetch the KIE container configurations. If not provided, the ID is automatically generated.
+
+|`org.kie.server.user`
+|String
+|`kieserver`
+|The user name used to connect with the {KIE_SERVER} from the controller, required when running in managed mode. Set this property in Business Central system properties. Setting this property is required when using a controller.
+
+|`org.kie.server.pwd`
+|String
+|`kieserver1!`
+|The password used to connect with the {KIE_SERVER} from the controller, required when running in managed mode. Set this property in Business Central system properties. Setting this property is required when using a controller.
+
+|`org.kie.server.token`
+|String
+|N/A
+|This property enables you to use a token-based authentication between the controller and the {KIE_SERVER} instead of the basic user name/password authentication. The controller sends the token as a parameter in the request header. Note that long-lived access tokens are required as the tokens are not refreshed.
```

Comment:
Instead of "This property enables you to use a token-based authe....", can we say "A property that enables you to use a token-based authe..." for consistency sake?  
Same for `org.kie.server.controller.token`

Reply:
fixed